tests/common/plugins/custom_skipif/__init__.py

```diff
@@ -0,0 +1,217 @@
+import multiprocessing
```

Comment:
As a general comment, all of this code should not be in `__init__.py`

Generally when debugging or developing, we are not expecting to see logic in the init other than some imports or globals. I would say its good code organization to either put these functions in their own file or include them with the class definition of `CustomSkipIf`

Reply:
will move to another place